src/Loop.zig

```diff
@@ -300,11 +300,8 @@ pub fn handleEventGeneric(self: anytype, vx: *Vaxis, cache: *GraphemeCache, Even
                 },
                 .winsize => |winsize| {
                     vx.state.in_band_resize = true;
-                    switch (builtin.os.tag) {
-                        .windows => {},
-                        // Reset the signal handler if we are receiving in_band_resize
-                        else => Tty.resetSignalHandler(),
-                    }
+                    // Reset the signal handler if we are receiving in_band_resize
+                    Tty.resetSignalHandler();
```

Comment:
We need to kip this switch unless you also add a `resetSignalHandler` to the `WindowsTty`. Can you remove this hunk?

Reply:
Done.